tests/backend/test_tf_ops.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r""" Tests for the file tfbackend/ops.py"""
+
+import pytest
+
+import numpy as np
+import tensorflow as tf
+
+from strawberryfields.backends.tfbackend.ops import reduced_density_matrix
+
+@pytest.mark.backends("tf", "fock")
+class TestTFOps:
+    """Testing for tfbackend/ops.py"""
```

Comment:
Since this test is to explicitly test the file `tfbackend/ops.py`, it feels a little weird to mark this test for the Fock backend as well.

Especially considering that if the developer doesn't have TF installed, this entire test file will be skipped anyway.

Reply:
sorry, just an oversight on my part, thanks for catching